test/groovy/DockerExecuteTest.groovy

```diff
@@ -27,34 +34,101 @@ class DockerExecuteTest extends BasePiperTest {
 
     int whichDockerReturnValue = 0
     def bodyExecuted
+    def containerName
 
     @Before
     void init() {
         bodyExecuted = false
         docker = new DockerMock()
+        JenkinsUtils.metaClass.static.isPluginActive = {def s -> new PluginMock(s).isActive()}
```

Comment:
Why not: 
`binding.setVariable('Jenkins', [instance: [pluginManager: [plugins: [new PluginMock('docker-workflow')]]]])`

Reply:
Because now the method has been moved to a utility class. The binding works only for the class we are testing `dockerExecute` in this case.  So it is necessary to replace the method itself.

Reply:
But in ArtifactSetVersionTest.groovy it works

Reply:
as far as I see, its unused in  `ArtifactSetVersionTest`

Reply:
removed the unused method binding